products/godot.md

```diff
@@ -19,15 +19,22 @@ auto:
 
 # Do not forget to remove the link after the first patch release.
 # For 3.x releases, they get deprecated on 3.x+1 release
-# 3.x will turn to LTS with the 3.6 release.
+# 3.x will turn to LTS with the 3.5 release.
```

Comment:
In the [docs](https://docs.godotengine.org/en/stable/about/release_policy.html#release-support-timeline) they wrote:
> Godot 3.6 (3.x, LTS)
> Godot 3.5

On the [start page](https://godotengine.org/) they have:
> Download LTS 3.5.3

I suggest for adding `lts: true` for cycle 3.5. WDYT?

Reply:
I agree

Reply:
Update done.